model/personconstructor.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
```

Comment:
Just a tip that it's best to leave off "constructor" in your module name. By virtue of it being a model, it should be implied that the person module is a constructor without explicitly stating it.